{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Ansible Automation Platform 2",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/a:redhat:ansible_automation_platform:2"
    },
    {
      "product_name": "Red Hat Certification for Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "rhcertification/redhat-certification-baremetal",
      "cpe": "cpe:/a:redhat:certifications:1::el8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Out of support scope",
      "package_name": "python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "python-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "python-s3transfer",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Out of support scope",
      "package_name": "resource-agents",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "container-tools:rhel8/python-podman",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "fence-agents",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "gimp:flatpak/python2-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "inkscape:flatpak/python2-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "python27:2.7/python2-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "python27:2.7/python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "python3.11-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "python3.11-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "python39:3.9/python3x-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "python39:3.9/python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "python-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Affected",
      "package_name": "fence-agents",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Not affected",
      "package_name": "python3.11-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Affected",
      "package_name": "python3.11-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Not affected",
      "package_name": "python-pip",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 3.11",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/a:redhat:openshift:3.11"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift4/baremetal-hardware-event-proxy-rhel8",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift4/cnf-tests-rhel8",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Container Platform 4",
      "fix_state": "Affected",
      "package_name": "openshift4/ztp-site-generate-rhel8",
      "cpe": "cpe:/a:redhat:openshift:4"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-api-server-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-artifact-manager-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-cache-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-persistenceagent-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-scheduledworkflow-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenShift Data Science (RHODS)",
      "fix_state": "Affected",
      "package_name": "rhods/odh-ml-pipelines-viewercontroller-rhel8",
      "cpe": "cpe:/a:redhat:openshift_data_science"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.2",
      "fix_state": "Out of support scope",
      "package_name": "python-botocore",
      "cpe": "cpe:/a:redhat:openstack:16.2"
    },
    {
      "product_name": "Red Hat OpenStack Platform 17.1",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/a:redhat:openstack:17.1"
    },
    {
      "product_name": "Red Hat Quay 3",
      "fix_state": "Affected",
      "package_name": "quay/quay-rhel8",
      "cpe": "cpe:/a:redhat:quay:3"
    },
    {
      "product_name": "Red Hat Satellite 6",
      "fix_state": "Affected",
      "package_name": "python-urllib3",
      "cpe": "cpe:/a:redhat:satellite:6"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Will not fix",
      "package_name": "rh-python38-python-pip",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Will not fix",
      "package_name": "rh-python38-python-urllib3",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    },
    {
      "product_name": "Red Hat Storage 3",
      "fix_state": "Affected",
      "package_name": "graphite-web",
      "cpe": "cpe:/a:redhat:storage:3"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2023-10-04T00:00:00Z",
  "bugzilla": {
    "description": "Cookie request header isn't stripped during cross-origin redirects",
    "id": "2242493",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2242493"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "5.9",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:H/PR:H/UI:N/S:U/C:H/I:H/A:N",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-200",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2023-43804",
  "document_distribution": "",
  "details": [
    "urllib3 is a user-friendly HTTP client library for Python. urllib3 doesn't treat the `Cookie` HTTP header special or provide any helpers for managing cookies over HTTP, that is the responsibility of the user. However, it is possible for a user to specify a `Cookie` header and unknowingly leak information via HTTP redirects to a different origin if that user doesn't disable redirects explicitly. This issue has been patched in urllib3 version 1.26.17 or 2.0.5."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-43804\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-43804"
  ]
}